docs/changelog.md

```diff
@@ -65,7 +65,7 @@ compile "org.janusgraph:janusgraph-core:0.6.0"
 * Apache Lucene 8.6.0
 * Apache Solr 7.7.2, 8.5.2
 * Apache TinkerPop 3.4.10
-* Java 1.8
+* Java 8, 11
```

Comment:
I'm just not sure that we can put `11` here if it is an experimental feature. I personally didn't test JanusGraph compatibility with Java 11 and we didn't execute all JanusGraph tests against Java 11 yet. Thus, I can't tell if we really tested JanusGraph compatibility with Java 11, unless you did it.
That said, I'm not against adding Java 11 compatibility here, it's just thoughts I have.

Another thought I have is that we should probably be consistent with Java versions here. If we change `Java 1.8` to `Java 8`, we should probably change it everywhere in this file. That said, it's more like a nitpick.

Reply:
I will keep just java 8 as support.